index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="refresh" content="0;url=demo/" />
+    <title>highcharts-chart</title>
+  </head>
+  <body>
+    <!--
+        Visit demo/index.html to see live examples of your element running.
+        This page will automatically redirect you there when run in the browser
+        with `polymer serve`.
+      -->
+  </body>
```

Comment:
Well for now, if that doesn't happen, say... github pages, which needs to work! We can do:
```suggestion
    <script>window.location = "demo/"</script>
  </body>
```